autoload/ale.vim

```diff
@@ -109,6 +109,7 @@ function! s:Lint(buffer, should_lint_file, timer_id) abort
     \   || (a:should_lint_file && filereadable(expand('#' . a:buffer . ':p')))
 
     call ale#engine#RunLinters(a:buffer, l:linters, l:lint_file)
+    call ale#code_lens#Request(a:buffer)
```

Comment:
I'm not sure this is the right way — what I need is to make requests in the same cases we do requests diagnostics (also after we synced changed to LSP).